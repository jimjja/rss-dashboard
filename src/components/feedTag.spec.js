import React from "react";
import renderer from "react-test-renderer";
import FeedTag from "./feedTag";

describe("Feed Tag component", () => {
  it("should match snapshot", () => {
    const props = {
      url: "sample url",
      isSelected: false,
      name: "Feed Tag",
      id: 1,
      onFeedDelete: () => {},
      onFeedClick: () => {},
      onFeedUpdate: () => {}
    };
    const tree = renderer.create(<FeedTag {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
