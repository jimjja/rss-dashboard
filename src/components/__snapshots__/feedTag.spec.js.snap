// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed Tag component should match snapshot 1`] = `
<div
  id={1}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  style={
    Object {
      "background": "#6e2a9e",
      "border": "1px solid #6e2a9e",
      "borderRadius": "30px",
      "color": "white",
      "cursor": "pointer",
      "display": "inline-block",
      "fontSize": "14px",
      "margin": "5px",
      "padding": "5px",
    }
  }
  title="sample url"
>
  <div
    onDoubleClick={[Function]}
  >
    <span
      onClick={[Function]}
      style={
        Object {
          "marginRight": ".3em",
        }
      }
      type="button"
    >
      Feed Tag
    </span>
    <span
      onClick={[Function]}
      type="button"
    >
      <i
        className="fa fa-times-circle fa-1x  "
        name="times-circle"
        size="1x"
        spin={false}
      />
    </span>
  </div>
</div>
`;
